### DISCUSSION TITLE
Guava variants are reported wrongly as updates

### DISCUSSION THREAD

**Discussion by savvasdalkitsis** [renovatebot/renovate#22740] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Renovate mend reports updates to the guava dependency updates on the wrong variant.

Example here: https://github.com/savvasdalkitsis/uhuruphotos-android/pull/305

`com.google.guava:guava:32.0.0-android` should not be upgraded to the `com.google.guava:guava:32.0.1-jre` variant

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
https://github.com/renovatebot/renovate/blob/22169819b866ea46fc00562cf9c8961545500133/lib/modules/versioning/regex/readme.md?plain=1#L14-L25
